plugins/jobtype/src/azkaban/jobtype/HadoopJavaJob.java

```diff
@@ -121,8 +121,27 @@ protected String getJVMArguments() {
     classPath.add(HadoopConfigurationInjector.getPath(getJobProps(),
         getWorkingDirectory()));
 
-    List<String> typeClassPath =
-        getSysProps().getStringList("jobtype.classpath", null, ",");
+    // merging classpaths from plugin.properties
+    mergeClassPaths(classPath,
+      getJobProps().getStringList("jobtype.classpath", null, ","));
+    // merging classpaths from private.properties
+    mergeClassPaths(classPath,
+      getSysProps().getStringList("jobtype.classpath", null, ","));
+
+    List<String> typeGlobalClassPath =
```

Comment:
Please note that we are not introducing this new param, it is already there (refer L141-L151). We cannot drop this param due to backward compatibility.